docs/articles_en/get-started/install-openvino/install-openvino-npm.rst

```diff
@@ -0,0 +1,62 @@
+.. {#openvino_docs_install_guides_installing_openvino_npm}
+
+Install Intel® Distribution of OpenVINO™ Toolkit from npm Registry
+==================================================================
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Windows, Linux, and
+                 macOS operating systems, using the npm registry.
+
+
+.. note::
+
+   Note that the npm distribution:
+
+   * offers the JavaScript API only
+   * is dedicated to users of all major OSes: Windows, Linux, and macOS
+     (all x86_64 / arm64 architectures)
+   * Windows and Linux do not offer support for GNA and NPU inference
```

Comment:
GNA is removed
NPU should be in JS (as per initial agreement, @vishniakov-nikolai please, confirm)

Reply:
Yes, correct. NPU supports in Node.js bindings, GNA should be removed.